client/src/entry/analysis/modules/Analysis.vue

```diff
@@ -25,10 +26,6 @@ const showPanels = computed(() => {
     return true;
 });
 
-const showActivityBar = computed(() => {
-    return userStore.toggledActivityBar;
```

Comment:
Might be worth porting to typescript. Was this broken because it should have been `return userStore.toggledActivityBar.value` ?

Reply:
I tried first putting .value, but it failed with "undefined does not have a value" :thinking:

Reply:
I will convert it to typescript too :+1:

Reply:
Done! 
Don't ask me why, but after renaming and converting to typescript, reactivity works, so I restored the computed property. Probably just exposing it in the store was the only issue here, but I could have sworn that it was not working just with that :shrug: